boms/cloud-lts-bom/pom.xml

```diff
@@ -58,7 +58,7 @@
     <google.cloud.bigtable.version>1.21.2</google.cloud.bigtable.version>
     <google.cloud.bigtable.client.version>1.19.1</google.cloud.bigtable.client.version>
     <google.autovalue.version>1.8.1</google.autovalue.version>
-    <google.api.client.version>1.31.3</google.api.client.version>
+    <google.api.client.version>1.31.3-sp.1</google.api.client.version>
```

Comment:
This version of google-api-client cannot build due to bintray problem. Therefore the modify-build-file approach cannot test it.

https://github.com/GoogleCloudPlatform/cloud-opensource-java/pull/1982#issuecomment-832056830